### PR TITLE
[Release] beta.1

### PR DESCRIPTION
**Pull request by quantum9Innovation**
- Major aesthetic improvements, mainly responsive design (945daba27c66a1d176d62ef0c555d74c8f1751a2) and support for iOS/Apple devices running WebKit/Safari browser engines (44d409b7c5f862507ad1e58dfea2faf60733c91c)
- Several important fixes, including consistent word generation (67954c4fc6f9d353eb61b2b5f2875999ed8412d9) and greater safety period enforcement (ec3d70aa81bf47450e2e7f6ae96684e490621210)
- Big updates: dark mode (e1f7fa4aa1192ff70300219179907a9ef85dd213) and combined spellcheck and frequency dictionaries (5111b348a2474ade41a2f01a9d725b880ca68bcc)

### CODE REVIEW

**Review by Quantalabs — CHANGES_REQUESTED** [review]
This is too big for one PR. Split into multiple smaller PRs.

**Comment by quantum9Innovation** [comment]
@checkly good, waiting on review from @Quantalabs before the official merge

**Comment by quantum9Innovation** [comment]
Change of plans, this is being backlogged as @Quantalabs won't be able to review until late next week. Meanwhile, this PR will merge into `main` since there seem to be no errors, and @Quantalabs will review from 9fc59afbd37d2eb1509f949df6ab83d59efb67b4 (will be tagged as `beta.1`) in a new discussion.

**Pull request merged**
